test/models/vehicle_test.rb

```diff
@@ -21,6 +21,34 @@ module Models
   class VehicleTest < Minitest::Test
     include Rack::Test::Methods
```

Comment:
not related to this PR but I just saw this for the first time `Rack::Test::Methods`, what do we do with it?

Reply:
I don't know :/

Reply:
@braktar ?

Reply:
Probably a copy/paste from another test file

Reply:
:facepalm: ffs